packages/dashboard/src/services/useAnonRoute.js

```diff
@@ -13,5 +11,5 @@ const fetcher = (url, router) => {
 export default function useAnonRoute() {
   const router = useRouter();
 
-  return useSWR(`${prefix}/api/user`, (url) => fetcher(url, router));
```

Comment:
removed because when running dashboard from server domain, requests were hitting portal domain and cookie did not authenticate them, we need them to hit the domain that we are currently on